### ISSUE TITLE
Unknown error while attempting to screenshot

### ISSUE THREAD

**Issue by pratu04** [RedSiege/EyeWitness#580]
Eyewitness shows Unknown error while attempting to screenshot this error on a URL I think ideally it shows this if there is no website present right?
Although it is working fine for other domains but for one of them it is showing this Unknown error while attempting to screenshot although there is a website running on it as I randomly checked manually.

Attached are screenshots.
![eyewitness01](https://user-images.githubusercontent.com/56386429/186657270-587d94ab-737c-4187-8dfd-412289b8e79c.png)
![eyewitness02](https://user-images.githubusercontent.com/56386429/186657282-cd5e6962-bd4b-4d10-8b4e-40ac2b6701f0.png)

**Comment by Relkci** [comment]
Can you check if the screenshots folder contains a screenshot of the host despite it reporting it failed?

I have a thought it may be urlib failing while the screen capture completed.

**Comment by tcpiplab** [comment]
Same error for me on Kali Linux. No screenshots in the directory either.

**Comment by Relkci** [comment]
anything special about that https service -- ? client cert / expired cert / revoked ca, etc?

Does this happen on all https?  Does it happen on http?

**Comment by Relkci** [comment]
no response - closing

**Comment by binarytrails** [comment]
in my case it was using `--proxy-ip 127.0.0.1 --proxy-port 1080 --proxy-type socks5` same situation, it creates images with names but gives `Unknown error while attempting to screenshot` and only target under `Web Request Info` no headers

**Comment by Tower450** [comment]
@Relkci having same issue but screenshots works and are present in folder but reports fails with `Unknown Error`

**Comment by Relkci** [comment]
Interesting.  There was a big code update late last year-- can you confirm you're using the current build from the github repo?

**Comment by Tower450** [comment]
`eyewitness/kali-rolling,now 20230525.1+git20230720-0kali4 amd64` probably because it's an old version but also `master` has the same behaviour

**Comment by binarytrails** [comment]
@Relkci In my case, I tried both this one and latest. Latest with the proxy settings [1] had a `socket error or timeout!` and `Version: 20230525.1+git20230720-0kali4`  ran creating screenshots files but doing `Unknown error while attempting to screenshot` error at `Web Request Info` field in html report.
```
# [1]
$ git log -1                                      
commit c3286fb71406e0b55b85a7c38dac76ee50b0f955 (HEAD -> master, origin/master, origin/HEAD)
Author: Tim Medin <tim@redsiege.com>
Date:   Mon Jan 5 11:58:46 2026 -0600

    fix kernel version retrieval. Fixes #714

$ sudo bash setup/setup.sh

[*] ╔══════════════════════════════════════════════════════╗
[*] ║        EyeWitness Setup (Virtual Environment)        ║
[*] ║                                                      ║
[*] ║  Production-ready installation using Python virtual  ║
[*] ║  environments to avoid PEP 668 and system conflicts  ║
[*] ╚══════════════════════════════════════════════════════╝

[+] Running with root privileges
[*] Detecting system information...
[*] Detected OS: kali 2025.2
[*] Detected Architecture: x86_64
[*] Checking Python installation...
[+] Python 3.13 found
[*] Starting EyeWitness installation...
[*] Installing system dependencies...
Hit:1 http://http.kali.org/kali kali-rolling InRelease
Reading package lists... Done
Reading package lists... Done
...
[+] Python dependencies installed
[*] Verifying Python package installations...
✓ selenium
✓ netaddr
✓ psutil
✓ pyvirtualdisplay
[+] All Python packages verified
[*] Testing EyeWitness installation...
[+] EyeWitness installation test passed

[+] ✓ EyeWitness installation completed successfully!

[*] USAGE:
[*] 1. Activate virtual environment: source eyewitness-venv/bin/activate
[*] 2. Run EyeWitness: python Python/EyeWitness.py [options]
[*] 3. Deactivate when done: deactivate

[*] TEST INSTALLATION:
[*] source eyewitness-venv/bin/activate
[*] python Python/EyeWitness.py --single https://example.com

[*] Virtual environment located at: /home/kali/workdir/packages/EyeWitness/eyewitness-venv
[*] Visit https://www.redsiege.com for more Red Siege tools

$ python3 Python/EyeWitness.py --threads 1 --timeout 30 --single http://10.10.10.10 --proxy-ip 127.0.0.1 --proxy-port 1080 --proxy-type socks5 --no-dns
[proxychains] config file found: /etc/proxychains4.conf
[proxychains] preloading /usr/lib/x86_64-linux-gnu/libproxychains.so.4
[proxychains] DLL init: proxychains-ng 4.17
[proxychains] Dynamic chain  ...  127.0.0.1:1080  ...  127.0.0.1:51101 <--socket error or timeout!
[proxychains] Dynamic chain  ...  127.0.0.1:1080  ...  127.0.0.1:51101 <--socket error or timeout!
[proxychains] Dynamic chain  ...  127.0.0.1:1080  ...  127.0.0.1:51101 <--socket error or timeout!
^C
```
